src/translations/locale/locale.en-US.json

```diff
@@ -0,0 +1,4 @@
+{
+  "validation.email.invalid": "Email is not correct",
+  "validation.required": "Field is required"
+}
```

Comment:
So basically, all we have to do now is add translations here and then map them like you did in `mapValidationMessagesToLocale` right?